admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -0,0 +1,372 @@
+.. _upgrade:
+
+============================================================
+Upgrade from Archivematica |previous_version|.x to |release|
+============================================================
+
+*On this page:*
+
+* :ref:`Create a backup <create-backup>`
+* :ref:`Upgrade Ubuntu package install <upgrade-ubuntu>`
+* :ref:`Upgrade CentOS/Redhat package install <upgrade-centos>`
```

Comment:
We should say `Red Hat`, I've just realized!
I've filed a bug: https://github.com/artefactual/archivematica-docs/issues/109.

Ignore for now, we can fix later.